vehicle/mkv/software/air_control/air.c

```diff
@@ -1,499 +1,433 @@
+#include <avr/interrupt.h>
+#include <stdlib.h>
+
+#include "libs/gpio/api.h"
+#include "libs/timer/api.h"
+
+#include "air_config.h"
+#include "bsp/bsp.h"
+#include "bsp/timer.h"
+#include "vehicle/mkv/software/air_control/can_api.h"
+
+/* TODO
+ *
+ * - Verify initial conditions and correct/fault conditions of all GPIOs
+ * - In FAULT state, should we clear PRECHARGE_CTL and AIR_N_LSD?
+ * - Should we be able to recover from any faults? Which ones? How?
+ */
+
+enum State {
+    INIT = 0,
+    IDLE,
+    SHUTDOWN_CIRCUIT_CLOSED,
+    PRECHARGE,
+    TS_ACTIVE,
+    DISCHARGE,
+    FAULT,
+};
+
+enum FaultCode {
```

Comment:
Replaced them with AIR_X. Regarding integrating to CAN API, I agree it would be a good idea, but maybe we can do that post-wheels spinning. Definitely crossed my mind though.